language/stdlib/modules/libra_time.move

```diff
@@ -39,4 +38,9 @@ module LibraTimestamp {
     public fun now_microseconds(): u64 acquires CurrentTimeMicroseconds {
         borrow_global<CurrentTimeMicroseconds>(0xA550C18).microseconds
     }
+
+    // Helper function to determine if the blockchain is at genesis state.
+    public fun is_genesis(): bool {
+        !::exists<Self::CurrentTimeMicroseconds>(0xA550C18)
```

Comment:
I think it would be much  more intuitive to move this to  `LibraBlock` and use `height == 0`. It looks like we may have to do some reordering of `lib.rs` to make this work, but it seems doable.